File: src/main/java/io/github/svegon/capi/ClientAPI.java
package io.github.svegon.capi;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public final class ClientAPI {
    public static Logger LOGGER = LogManager.getLogger("Client API");

    private static final ClientAPI INSTANCE = new ClientAPI();

    private ClientAPI() {
    }

    public static ClientAPI getInstance() {
        return INSTANCE;
    }
}

File: src/main/kotlin/io/github/svegon/mclientapi/ClientAPI.kt
package io.github.svegon.mclientapi

import io.github.svegon.mclientapi.ClientAPI
import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

object ClientAPI {
    var LOGGER: Logger = LogManager.getLogger("Client API")

    val instance: ClientAPI = ClientAPI()
}
